langchain/llms/llamacpp.py

```diff
@@ -103,6 +103,9 @@ class LlamaCpp(LLM):
     streaming: bool = True
     """Whether to stream the results, token by token."""
 
+    verbose: bool = True
```

Comment:
is this the default? or is it default false and this is actually a change from existing behavior?

Reply:
I don't know why, but yes, the default is set to True in the API.

https://llama-cpp-python.readthedocs.io/en/latest/api-reference/